### DISCUSSION TITLE
Stripe Payment Integration

### DISCUSSION THREAD

**Discussion by usmanisofficial** [cytech/BillingTrack-Wired#17] [General]
Hi, 
I have added Public Key and Secret Key in Payment setting, but when I clicked on the Pay with Stripe button, it did not work. 
I just copied PK and SK from stripe dashboard.

**Comment by cytech** [comment]
by "it did not work" does that mean that no Stripe payment dialog was displayed?

**Reply by usmanisofficial** [reply]
Yes, it is getting a 302.
Here is the URL of the Public Invoice.
https://billing-track.coduko.com/client_center/invoice/FzdCCxdsoGuxlNhmx1kQIwyKTcYgJIKi

**Comment by cytech** [comment]
browser devtools throwing a javascript error on page load:
Uncaught TypeError: Failed to execute 'getComputedStyle' on 'Window': parameter 1 is not of type 'Element'
not sure why at this point...

**Comment by cytech** [comment]
there may have been a conflict with the BS5 version of admin-lte, particularly if you rebuilt the app.js.
i uploaded  a new version v6.1.2 with the latest build of admin-lte v4
